botcity/maestro/impl/v1.py

```diff
@@ -114,7 +114,7 @@ def message(self, email: List[str], users: List[str], subject: str, body: str,
                 )
 
     def create_task(self, activity_label: str, parameters: Dict[str, object],
-                    test: bool = False) -> model.AutomationTask:
+                    test: bool = False, **kwargs) -> model.AutomationTask:
```

Comment:
I believe we will also need *args to avoid issues if called without passing the arg name.